### PR TITLE
[Components with more than 1 service]  LPS-190897 Refactor AMImageProcessorImpl component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related: https://liferay.atlassian.net/browse/LPS-190897

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:lima

File: modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/processor/AMImageProcessorImpl.java [hunk 1]
```diff
@@ -14,178 +14,18 @@
...
 import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Adolfo Pérez
  */
 @Component(
 	property = "model.class.name=com.liferay.portal.kernel.repository.model.FileVersion",
-	service = {AMImageProcessor.class, AMProcessor.class}
+	service = AMImageProcessor.class
```
Comment (rafaprax) [inline modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/processor/AMImageProcessorImpl.java:27]: Hey @Valmir-unicap, I think you do not need to keep the property `model.class.name` for this service. Can you please check that and remove it? Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:lima

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Review by MrJohn1911 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**